src/components/atoms/SearchBar/index.jsx

```diff
@@ -108,13 +108,28 @@ class SearchBar extends React.Component {
 }
 //#endregion
 
-//#region > PropTypes
-SearchBar.propTypes = {};
+//#region > Redux Mapping
+const mapStateToProps = (state) => ({
+  allRegisteredUsernames: state.general.allRegisteredUsernames,
+});
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    allUsernames: () => dispatch(getAllPageUrlsAction()),
+  };
+};
 //#endregion
 
 //#region > Exports
-//> Default Class
-export default withRouter(SearchBar);
+/**
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ * Got access to the history object’s properties and the closest
+ * <Route>'s match.
+ */
```

Comment:
```suggestion
/**
 * Provides its connected component with the pieces of the data it needs from
 * the store, and the functions it can use to dispatch actions to the store.
 *
 * Got access to the history object’s properties and the closest
 * <Route>'s match.
 */
```